### ISSUE TITLE
mkl-platform-redist on Sbt

### ISSUE THREAD

**Issue by yukoba** [bytedeco/javacpp-presets#823]
Hello. I can use mkl-platform-redist on Maven. But I cannot use mkl-platform-redist on Sbt.

I added these to build.sbt.
```
libraryDependencies += "org.bytedeco" % "mkl-platform" % "2019.5-1.5.2"
libraryDependencies += "org.bytedeco" % "mkl-platform-redist" % "2019.5-1.5.2"
```

The error message is this.
```
Exception in thread "main" java.lang.UnsatisfiedLinkError: no jnimkl_rt in java.library.path
```

I intentionally not installed Intel MKL 2019.5.
What should I do to use mkl-platform-redist on Sbt?

**Comment by saudet** [comment]
That should work just fine. What does it print to the console if you set the "org.bytedeco.javacpp.logger.debug" system property to "true"?

Also, what is the output of `sbt "inspect tree clean"`?

**Comment by yukoba** [comment]
I pasted logger.txt and sbtlog.txt to https://gist.github.com/yukoba/52c7f9746f1981fe8bfbb2c56444aff4 .

The meaning of the Japanese error message "クライアントは要求された特権を保有していません。" is "a required privilege is not held by the client."

My environment.
- OS: Windows 10 Japanese
- Intel MKL 2019.4: Installed
- Intel MKL 2019.5: Not installed

**Comment by saudet** [comment]
`mkl-platform-redist` isn't showing up as a dependency in your build, and it's not getting added to the class path either. This looks like an issue with sbt. Could you report this issue upstream?

**Comment by saudet** [comment]
Until sbt fixes this, adding directly the dependencies of `mkl-platform-redist` might work:
http://repo2.maven.org/maven2/org/bytedeco/mkl/2019.5-1.5.2/

**Comment by yukoba** [comment]
I reported to https://github.com/sbt/sbt/issues/5268 .
Also I uploaded the test code to https://github.com/yukoba/MklTest .

**Comment by yukoba** [comment]
Sbt 1.3 changed from Apache Ivy to Coursier.
I found that if I disable Coursier by ```ThisBuild / useCoursier := false``` this problem does not happen.

**Comment by saudet** [comment]
This has apparently been fixed: https://github.com/sbt/sbt/issues/5046#issuecomment-572310336. @yukoba Can you confirm?

**Comment by yukoba** [comment]
This is not fixed in sbt 1.3.7 yet. Should I reopen https://github.com/sbt/sbt/issues/5268 ?

**Comment by yukoba** [comment]
Sorry. I found the solution. I have to delete Coursier cache. So, this is fixed in sbt 1.3.7. I will close this.
